proxylib/npds/client_test.go

```diff
@@ -56,17 +57,21 @@ var resources = []*cilium.NetworkPolicy{
 
 // UpsertNetworkPolicy must only be used for testing!
 func (cs *ClientSuite) UpsertNetworkPolicy(c *C, s *envoy.XDSServer, p *cilium.NetworkPolicy) {
-	comp := completion.NewCompletion(nil, func(err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Why 5 seconds and not 10 for example?

Reply:
No reason, most likely I copied this from somewhere else in the test. 5 seconds should be long enough given that both client and server run locally within the test.